### DISCUSSION TITLE
Generating a scree plot of principal components for PC adjustment

### DISCUSSION THREAD

**Discussion by ghost** [PGScatalog/pgsc_calc#259] [General]
I ran PGSC_calc using the recommended 1kG+HGDP reference panel to adjust the polygenic scores for genetic similarity. I wanted to generate a scree plot to justify my choice in the number of PCs used for adjustment. Are the eigenvalues for the PCs stored somewhere in the tool's working directory? I tried looking under the fraposa sub-directory and wasn't able to find it in there. Thanks for your help!

**Comment by smlmbrt** [comment]
@rnatara5 unfortunately the tool we use for the PCA derivation (https://github.com/PGScatalog/fraposa_pgsc) doesn't currently output that information (only the singular values). If you are able to modify the code to output the eigenvalues we could build that into the pipeline outputs easily.

**Reply by GreenGilad** [reply]
Unless I am missing something the eigenvalues are the square of the singular values - so if you have the singular values information it is sufficient
